spec/services/exporter/service_spec.rb

```diff
@@ -71,20 +71,36 @@ def extract_zipped_exports(buffer)
     context "when emailing the user" do
       it "delivers one email" do
         service = valid_instance(article.user)
-        service.export(send_email: true)
+        service.export(send_to_admin: false)
         expect(ActionMailer::Base.deliveries.count).to eq(1)
       end
 
+      it "delivers one email to the user's email" do
+        service = valid_instance(article.user)
+        service.export(send_to_admin: false)
+        expect(ActionMailer::Base.deliveries.last.to.first).to eq article.user.email
+      end
+
       it "delivers an email with the export" do
         service = valid_instance(article.user)
-        zipped_export = service.export(send_email: true)
+        zipped_export = service.export(send_to_admin: false)
         attachment = ActionMailer::Base.deliveries.last.attachments[0].decoded
 
         exports = extract_zipped_exports(zipped_export)
         expect(exports).to eq(extract_zipped_exports(StringIO.new(attachment)))
       end
     end
 
+    context "when emailing an admin" do
+      it "delivers one email to the default admin email" do
+        admin_email = "admin@example.com"
+        SiteConfig.email_addresses = { default: admin_email }
```

Comment:
There might be a cleaner way of updating an ENV var within a test?

Reply:
I would use stubbing instead. You are setting env var that could bleed into other test.
```suggestion
        allow(SiteConfig).to receive(:email_addresses).and_return({ default: admin_email })
```